bhclub.html

```diff
@@ -52,7 +46,51 @@ <h1>Events</h1>
             <h1>Teams</h1>
         </section>
         <section id="contact">
-            <h1>Contact</h1>
+            <div>
+                <h1>Contact</h1>
+                <p>Have any questions? Feel free to reach out to us at <b>hello@brickhack.org</b></p>
+                <a class="btn-light" href="mailto:hello@brickhack.org">EMAIL US</a>
+                <p>You can also sign up to receive our BrickHack event newsletter here:</p>
+                <!-- MailChimp email form, copied from bh8 preregister -->
+                <div class="field-row-stacked">
+                    <div id="mc_embed_signup">
+                        <form action="https://coderit.us11.list-manage.com/subscribe/post?u=122b09a8cef4c1f3888af8e40&amp;id=4c1af7f783" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+                            <div id="mc_embed_signup_scroll">
+                                <div class="mc-field-group">
+                                    <div class="field-row">
+                                        <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" placeholder="name@example.com">
+                                        <button type="submit">SUBMIT</button>
+                                    </div>
+                                </div>
+                                <div id="mce-responses" class="clear">
+                                    <div class="response" id="mce-error-response" style="display:none"></div>
+                                    <div class="response" id="mce-success-response" style="display:none"></div>
+                                </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                                <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_122b09a8cef4c1f3888af8e40_4c1af7f783" tabindex="-1" value=""></div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+            <div>
+                <div id="line"></div>
+                <div id="footer">
+                    <div>
+                        <p>Rochester Institute of Technology</p>
+                        <p>1 Lomb Memorial Dr, Rochester, NY 14623</p>
+                    </div>
+                    <div id="socials">
+                        <span id="social-text">Connect with us!</span>
+                        <a class="social-link" href="https://discord.gg/gNJXjvzkuB" target="_blank">Discord</a>
+                        <a class="social-link" href="#" target="_blank">CampusGroups</a>
+                        <a class="social-link" href="#" target="_blank">GitHub</a>
+                        <a class="social-link" href="#" target="_blank">Twitter</a>
```

Comment:
Note that these are broken in Firefox; needs a [browser prefix](https://developer.mozilla.org/en-US/docs/Web/CSS/width). 
This _should_ be fixed automatically by Parcel but we also might need to [manually install PostCSS](https://parceljs.org/css.html#postcss). I thought that our SASS plugin would do this :(